aaelf64/aaelf64.rst

```diff
@@ -1260,16 +1260,32 @@ Linkers may optionally optimize instructions affected by relocation. Relocation
 
     ::
 
-      ADD x0, x1, 0   // eg. R_<CLS>_TLSLE_ ADD_TPREL_HI12
-      ADD x2, x2, 0   // or R_<CLS>_ADD_ABS_LO12_NC
+      ADD   x0, x1, 0   // eg. R_<CLS>_TLSLE_ADD_TPREL_HI12
+      ADD   x2, x2, 0   // or R_<CLS>_ADD_ABS_LO12_NC
 
       // after optimization:
 
-      MOV x0, x1
+      MOV   x0, x1
       NOP
 
   - The relocation ``R_<CLS>_ADR_PREL_PG_HI21`` may emit a MOV with zero immediate for undefined weak symbols.
 
+  - The following TLS relocations may be optimized if the symbol is a non-preemtable definition and the TLS offset fits in 16 bits:
```

Comment:
Typo preemtable with no p. To be consistent with the rest of the document I think non-pre-emptable, or definition that cannot be pre-empted.

Reply:
I'll fix